Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -7,17 +7,41 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
     /// The individual elements of the expression in an array
     internal var elements: [Element]
 
+    /// The operator of this expression
+    public var `operator`: Operator {
+        guard let first = elements.first, case Element.operator(let op) = first else {
+            fatalError("First element of the expression is not an operator.")
```

Comment:
If it’s possible to create a malformed expression like this, consider adding a test that this error is thrown, so it doesn’t accidentally fall out during a refactor down the road.

Reply:
Ah it's not actually possible to do that.. I'm just fatally erroring to avoid making this return an optional.